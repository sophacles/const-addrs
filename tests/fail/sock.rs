use const_addrs::sock;

fn main() {
    let _ = sock!("192.168.11:300");
    let _ = sock!("2001:db8::32::23:22");
}
