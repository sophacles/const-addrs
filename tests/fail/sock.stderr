error: invalid socket address syntax
 --> tests/fail/sock.rs:4:19
  |
4 |     let _ = sock!("192.168.11:300");
  |                   ^^^^^^^^^^^^^^^^

error: invalid socket address syntax
 --> tests/fail/sock.rs:5:19
  |
5 |     let _ = sock!("2001:db8::32::23:22");
  |                   ^^^^^^^^^^^^^^^^^^^^^
